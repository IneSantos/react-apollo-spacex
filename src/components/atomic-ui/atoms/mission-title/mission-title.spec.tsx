import React from "react";
import { render } from "@testing-library/react";
import MissionTitle from "./mission-ttile";

describe("MissionTitle atom tests", () => {
  test(`MissionTitle component render`, () => {
    const { container } = render(<MissionTitle name="This is a mission name" />);
    expect(container.firstChild).toMatchSnapshot();
  });
});
