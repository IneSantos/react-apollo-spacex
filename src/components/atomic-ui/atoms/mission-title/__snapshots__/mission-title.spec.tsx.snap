// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`MissionTitle atom tests MissionTitle component render 1`] = `
<h2
  class="title"
>
  This is a mission name
</h2>
`;
